src/main/java/gregtech/api/capability/impl/ItemFuelInfo.java

```diff
@@ -15,6 +15,6 @@ public ItemFuelInfo(ItemStack itemStack, int fuelRemaining, int fuelCapacity, in
     }
 
     public String getFuelName() {
-        return itemStack.getUnlocalizedName();
+        return itemStack.getTranslationKey();
     }
```

Comment:
Not sure why this change got carried into this PR, but I will make sure it is a non-issue by the time this PR is ready to be merged.